python/mpt/src/mpt/ethereum_tries.py

```diff
@@ -345,15 +346,94 @@ def from_data(cls, data: Dict[str, Any]) -> "EthereumTrieTransitionDB":
         instance.post_state_root = post_state_root
         return instance
 
-    def to_pre_state(self) -> State:
-        """Convert the pre-state trie to a State object."""
-        return self.to_state()
 
-    def to_post_state(self) -> State:
-        """Convert the post-state trie to a State object."""
-        state = State()
-        root_node = self.nodes[self.post_state_root]
-        self.traverse_trie_and_process_leaf(
-            root_node, b"", partial(self.set_account_from_leaf, state=state)
+@dataclass
+class PreState:
```

Comment:
arguably a nit but we don't need a dataclass for a single static method, wdyt?

Reply:
true